### ISSUE TITLE
QUESTION

### ISSUE THREAD

**Issue by EricSmith17** [djandiek/7dtd-bot-nodejs#1]
Can i make this to make bot connect to someones server ?
i want to make simple AI bot client that will defend my base for example

**Comment by djandiek** [comment]
Yes you can if you know the connection details. You would need the IP and telnet password of the server. Look in the config.json file as to where to make the changes.
